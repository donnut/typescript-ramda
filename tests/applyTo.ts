import R_applyTo = require('../ramda/dist/src/applyTo');

declare const string: string;
declare const string_to_number: (x: string) => number;

// @dts-jest:pass:snap
R_applyTo(string);
// @dts-jest:pass:snap
R_applyTo(string)(string_to_number);
// @dts-jest:pass:snap
R_applyTo(string, string_to_number);
