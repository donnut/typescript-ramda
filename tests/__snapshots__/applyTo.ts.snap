// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R_applyTo(string) 1`] = `"<R>(fn: (value: string) => R) => R"`;

exports[`R_applyTo(string)(string_to_number) 1`] = `"number"`;

exports[`R_applyTo(string, string_to_number) 1`] = `"number"`;
